substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -494,11 +493,7 @@ impl PalletCmd {
 
 							log::info!(
 								target: LOG_TARGET,
-								"Running  benchmark: {}.{}({} args) {}/{} {}/{}",
-								String::from_utf8(pallet.clone())
-									.expect("Encoded from String; qed"),
-								String::from_utf8(extrinsic.clone())
-									.expect("Encoded from String; qed"),
+								"Running  benchmark: {pallet_str}.{extrinsic_str}({} args) {}/{} {}/{}",
```

Comment:
```suggestion
								"Running  benchmark: {pallet_str}::{extrinsic_str}({} args) {}/{} {}/{}",
```
Noticed recently that it should be consistent with the print above.